platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/model/constants/AppConstant.java

```diff
@@ -0,0 +1,6 @@
+package com.mapbox.mapboxsdk.testapp.model.constants;
+
+public class AppConstant {
+
+    public final static int STYLE_VERSION = 9;
```

Comment:
This number is also in integers.xml. Is there a way – perhaps in unit tests or in a script that gets run in CI – to ensure that these versions are in sync?